docs/gettingstarted.rst

```diff
@@ -1,205 +1,65 @@
 
 .. code-block:: console
 
-   $ kubectl get all
-   NAME             CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
-   svc/kubernetes   10.0.0.1     <none>        443/TCP   3d
-
-
-Install reana-cluster cli tool
-------------------------------
-
-reana-cluster command line interface tool is not yet released
-in PyPI, so install it from our GitHub repotory:
-
-.. code-block:: console
-
-   $ pip install \
-     -e 'git+https://github.com/reanahub/reana-cluster.git@master#egg=reana-cluster'
-
-
-Initialize a REANA cluster
---------------------------
-
-Main function of reana-cluster command line tool is to
-initialize a working REANA cluster, ready to run workflows
-you submit to it using reana-client.
-
-In order to achieve this reana-cluster needs to know how
-REANA cluster should be set up; e.g. what versions of REANA
-components should be deployed and how the configuration of each
-component should be set up.
-
-reana-client expects to get information via REANA cluster
-specification file.
+   $ mkvirtualenv reana-cluster
+   $ pip install -e 'git+https://github.com/reanahub/reana-cluster.git@master#egg=reana-cluster'
 
-The specifications file is written in YAML syntax, but since
-reana-cluster is still work-in-progress the structure of
-REANA specifications file might change rapidly.
-We therefore suggest that you download our default
-configuration file from the root of our GitHub repository:
+4. Configure REANA cluster. You can download default ``reana-cluster.yaml`` configuration file:
 
 .. code-block:: console
 
    $ wget https://raw.githubusercontent.com/reanahub/reana-cluster/master/reana-cluster.yaml
 
-
-Default REANA cluster specifications file deploys latest
-released versions of all REANA components in their
-default configuration. Please note that default specifications file
-is intended for evaluation, not for production deployments.
-
-After downloading the specifications file it is just a matter of
-running `init` with reana-cluster:
+5. Start REANA cluster instance on Minikube:
 
 .. code-block:: console
 
    $ reana-cluster init
 
-
-Verify REANA components
------------------------
-
-You can verify that components deployed to REANA cluster are set up according
-to what is defined in REANA cluster specifications file `verify`:
+6. Check status of deployed REANA components:
 
 .. code-block:: console
 
    $ reana-cluster verify components
 
-
-Get information about a deployed REANA component
-------------------------------------------------
-
-To print component specific information, for example URLs that can
-be used to interact with the component run
-``reana-cluster get <COMPONENT_NAME>``.
-When REANA cluster is deployed on minikube ``get``-command returns
-an IP-address+port combination. This information can be used to construct URL
-to access component's API or user-interface in case component provides one.
+7. Delete REANA cluster. If you want to stop working on REANA, you can bring it down as follows:
```

Comment:
Saying:
> If you want to stop working on REANA, you can bring it down as follows

Sounds to me like it is reversible change (like, I stop working now and I come later to continue), but in fact it is not. Therefore, if were I am a new user, I would do the `reana-cluster down` command and I would be looking for the up command, then I would not find it and I would end up using `reana-cluster init` producing potentially a different setup if for instance `reana-cluster.yaml` was changed.

Reply:
Good point. Not sure what to do about this. We could highlight this in documentation and maybe add capitalized help text to `reana-cluster down --help`?

Reply:
This same thing applies to here as well
https://github.com/reanahub/reana-cluster/pull/38/files#diff-bb7aeded5e46a9d6e405411c85a2d47dR199